Code/ThirdParty/LivePP/init.cmake

```diff
@@ -0,0 +1,35 @@
+# LIVE++ Utils
+# NOTES: To not use github storage, you can download the Live++ SDK from the official website and put it in the same directory as this file.
+
+if(EZ_CMAKE_PLATFORM_WINDOWS)
+    set(EZ_3RDPARTY_LIVEPP_SUPPORT OFF CACHE BOOL "Enable Live++ support for Ez")
```

Comment:
in this code path it would be nice to check with CMake whether the Live++ binaries exist, which means that the user put them there themselves, and then enable EZ_3RDPARTY_LIVEPP_SUPPORT.

This can easily be done with CMake just with

```
if (EXISTS "${CURRENT_SOURCE_DIR}/something.lib")
```

You would probably need to set the CACHE variable with FORCE, to ensure it gets overridden, so that users who copy the libs in later still get the feature enabled.

Reply:
The preview of this review doesn't show it, but in the latest commit, I added a check for if the Live++ broker was in the directory. 

Would you want me to check for the DLL files?

Reply:
The preview of this review doesn't show it, but in the latest commit, I added a check for if the Live++ broker was in the directory. 

Would you want me to check for the DLL files?

Reply:
Oh I did not see that. But then you need to change this line to ON, because currently it is always switched to OFF.

Reply:
Also I would test for EZ_CMAKE_PLATFORM_WINDOWS_DESKTOP, to exclude UWP.

Reply:
Resolved.